cogs/roles.py

```diff
@@ -69,6 +69,7 @@ async def toggle_role(self, ctx, transaction: RoleTransaction,
         Assigns a single role to a user with no checks from a category of roles
         """
 
+        fcategory = f"{'categories' if len(categories) > 1 else 'category'}"
```

Comment:
format-string isnt needed, could just be
```python
fcategory = "categories" if len(categories) > 1 else "category"

Reply:
Resolved in 4837c90